be-java/src/commonTest/kotlin/lang/temper/be/java/JavaFunctionalTest.kt

```diff
@@ -58,15 +58,14 @@ abstract class JavaFunctionalTest(factory: Backend.Factory<JavaBackend>) : Funct
             packageName = QualifiedName.fromTemperPath(mainModuleName.sourceFile),
             module = mainModuleName,
         )
-        val runJava = if (test.runAsTest) RunAsTest else RunByExec(moduleInfo.entryQualifiedName)
+        val runJava = if (test.runAsTest) RunAsTest else RunByExec(moduleInfo.entryQualifiedName, bundled = true)
```

Comment:
Provide the flag we need for funtest running to continue as before. Might want to get rid of bundled entirely sometime, though.